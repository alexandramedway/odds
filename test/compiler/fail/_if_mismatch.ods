do if true then 42 else "hello"
