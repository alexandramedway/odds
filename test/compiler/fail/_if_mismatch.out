Analyzer Exception: Invalid attempt to use conditional with mismatched types Num and String
